routing/simple.html

```diff
@@ -0,0 +1,240 @@
+crossorigin=""></script>
+  <!--script src="https://code.jquery.com/jquery-2.2.4.min.js" integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script-->
+  <script src="http://code.jquery.com/jquery-3.5.1.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
+  <script src="../polyline/decode.js"></script>
+
+  <div style="width:100%">
+  <div id="map" style="height: 1050px"></div>
+  Base URL: <input id="baseurl" placeholder="http://localhost:8002/"></input></br>
+  Valhalla Token: <input id="token" rows="1" cols="30" maxlength="100" wrap="hard" placeholder="Enter token."></input></br>
+  Request Body:</br><textarea id="request_body" rows="5" cols="100" maxlength="500" wrap="hard">{"costing":"auto"}</textarea></br>
+  <button type="button" id="route">Route</button>
+
+  </div>
+
+  <script>
+    ////////////////////// INITIALIZATION ////////////////////////
+
+    //make a map using osm tiles
+    const map = L.map('map').setView([40.2, -76.6], 10);
+    const baseLayer = L.tileLayer('http://b.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+      maxZoom: 19,
+      attribution: '&copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributers'
+    });
+    baseLayer.addTo(map);
+
+    //state
+    let geojson = null;
+    let inputs = [];
+
+    /////////////////// REQUEST PROCESSING //////////////////////
+
+    //do all the requests
+    async function fetch(json_body) {
+      // parse the requests
+      let requests = null;
+      try {
+        requests = JSON.parse(json_body);
+        if(!Array.isArray(requests)) {
+          requests = [requests]
+        }
+      }
+      catch(e) {
+        alert('Invalid Json Request');
+      }
+
+      //create base url
+      let baseurl = document.getElementById('baseurl').value;
+      if (!baseurl)
+        baseurl = 'http://localhost:8002/';
+      baseurl += 'route?';
+      const token = document.getElementById('token').value;
+      if (token)
+        baseurl += 'access_token=' + token + '&json=';
+      else
+        baseurl += 'json=';
+      
+      //do them all
+      const responses = await Promise.all(
+        requests.map(async (request) => {
+          const url = baseurl + JSON.stringify(request);
+          const response = await $.getJSON(url);
+          return response;
+        })
+      );
+
+      //set the text box
+      return {requests, responses};
+    }
+
+    ////////////////////// RESPONSE PROCESSING /////////////////////////
+
+    //convert the result into geojson so its displayable
+    function build_geojson(responses) {
+      const fc = { type:'FeatureCollection', features: [] };
+
+      // for each response
+      responses.forEach( (response) => {
+        // pick a main color and then each alternate will be lighter
+        let color = Math.floor(Math.random()*16777215*.5);
```

Comment:
TODO: pipe the original request through here too and check for its color property if it has one